tests/unit/store/flask.spec.js

```diff
@@ -266,12 +266,12 @@ describe("store/flask", () => {
         await store.dispatch("flask/start_status_pinging");
       });
 
-      test("When start_status_pinging is dispatched, the status_ping_interval_id state does not change and setInterval is not called", async () => {
+      test("Given start_status_pinging is dispatched, When status_ping_interval_id state does not change, Then setInterval is not called", async () => {
```

Comment:
the only function called in this test is to dispatch start_status_pinging, so that must be the "When" section. I don't think there is any "Given"/"Arrange" setup for this test, the test just immediately starts by dispatching start_status_pinging.  Also, just FYI---If a spy does not alter the implementation of the method being spied on, then it does not need to be listed in the "Given" section since it is not affecting how the code behaves, it just allows us to assert things about the function being spied on

Reply:
You said "Given section is removed", but it does not look like anything changed...

Reply:
Sorry as there are so many comments, missed not at line `267` the `Given` is removed.